locking/gradle.lockfile

```diff
@@ -0,0 +1,70 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** GHSA-36p3-wjmg-h94x   
**Check Name:** Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection')   
**Severity:** UNKNOWN   
**Fixed Version:** 5.2.20, 5.3.18   
**Description:** Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection') in org.springframework.boot:spring-boot-starter-webflux.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://github.com/advisories/GHSA-36p3-wjmg-h94x